pages/invalid-team-invitation.js

```diff
@@ -0,0 +1,37 @@
+import { useRouter } from 'next/router'
+import Link from 'next/link'
+import { Typography, Result, Button } from 'antd'
+
+const { Paragraph, Text } = Typography
+
+const InvalidTeamInvitationPage = () => {
+  const router = useRouter()
+  const link = router.query.link
+
+  return (
+    <div>
+      <Result
+        status="error"
+        title="Invalid invitation link"
+        subTitle={
+          <div>
+            <Paragraph>The link you used: <Text style={{ overflowWrap: 'break-word' }}><a href={link}>{link}</a></Text></Paragraph>
```

Comment:
perhaps add " is not valid" somewhere here? I might be missing hte wider context of where this is displayed though.

Reply:
I forgot to add the screenshot 😱

Reply:
@mrsheepuk I've added a screenshot, let me know what you think.